kits/default/assets/sass/global/_variable.scss

```diff
@@ -506,38 +506,6 @@ $list-group-link-color:          #555 !default;
 $list-group-link-heading-color:  #333 !default;
```

Comment:
These Sass variables are required. They are part of Bootstrap panels and not Drupal panels. See http://getbootstrap.com/components/#panels. Thanks.